exercises/concept/census/.docs/instructions.md

```diff
@@ -0,0 +1,98 @@
+# Instructions
```

Comment:
I really, really like this story and their tasks! Well done 👍

Reply:
Thank you kindly!